.github/workflows/run_unit_test.yaml

```diff
@@ -0,0 +1,7 @@
+name: Run Unit Tests
+on: [push, pull_request]
+jobs:
+  run_unit_test:
+    uses: Cray-HPE/hms-build-scripts/.github/workflows/run_unit_test.yaml@develop
```

Comment:
Switch to the master branch when https://github.com/Cray-HPE/hms-build-scripts/pull/4 is merged